browser/js/reviews/review-item/review-item.html

```diff
@@ -0,0 +1,14 @@
+<!-- <div class="col-xs-6"> -->
+		<div class="well text-center" style="height: 150px">Movie image</div>
+		<div class="panel-body">
+			<div class="list-group">
+				<div class="list-group-item"><strong>{{review.title}}, {{review.movie}} by {{review.user}}</strong></div>
+				<div class="list-group-item">Created: {{review.dateCreated}}</div>
```

Comment:
We might need to format dateCreated on the front of back end because I think momentjs will return a really long ugly format. maybe use momentjs to do relative time for added sexiness: http://momentjs.com/docs/#/displaying/fromnow/